client/extensions/woocommerce/state/sites/payment-methods/test/reducer.js

```diff
@@ -9,12 +9,61 @@ import { expect } from 'chai';
 import reducer from '../../reducer';
 import { LOADING } from 'woocommerce/state/constants';
 import {
+	WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE,
+	WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE_SUCCESS,
+	WOOCOMMERCE_PAYMENT_METHOD_UPDATE,
+	WOOCOMMERCE_PAYMENT_METHOD_UPDATE_SUCCESS,
 	WOOCOMMERCE_PAYMENT_METHODS_REQUEST,
 	WOOCOMMERCE_PAYMENT_METHODS_REQUEST_SUCCESS,
-	WOOCOMMERCE_PAYMENT_METHOD_UPDATE_SUCCESS,
 } from 'woocommerce/state/action-types';
 
 describe( 'reducer', () => {
+	describe( 'WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE', () => {
+		it( 'should return the same state pased in', () => {
+			const siteId = 123;
+			const state = {};
+			const action = {
+				type: WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE,
+				siteId,
+				data: {},
+			};
+
+			const newState = reducer( state, action );
+			expect( newState ).to.eql( {} );
+		} );
+	} );
+
+	describe( 'WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE_SUCCESS', () => {
+		const siteId = 123;
+		const state = { [ siteId ]: {
+			paymentMethods: [ { id: 'bar', enabled: false } ]
+		} };
+		const action = {
+			type: WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE_SUCCESS,
+			siteId,
+			data: { id: 'bar', enabled: true },
+		};
+
+		const newState = reducer( state, action );
+		expect( newState[ siteId ] ).to.exist;
+		expect( newState[ siteId ].paymentMethods ).to.deep.equal( [ { id: 'bar', enabled: true } ] );
+	} );
+
+	describe( 'WOOCOMMERCE_PAYMENT_METHOD_UPDATE', () => {
```

Comment:
This is:  WOOCOMMERCE_PAYMENT_METHOD_UPDATE  the one above is:  WOOCOMMERCE_PAYMENT_METHOD_ENABLED_UPDATE